examples/src/main/java/io/grpc/examples/routeguide/RouteGuideServer.java

```diff
@@ -120,7 +120,7 @@ public static void main(String[] args) throws Exception {
   private static class RouteGuideService extends RouteGuideGrpc.RouteGuideImplBase {
     private final Collection<Feature> features;
     private final ConcurrentMap<Point, List<RouteNote>> routeNotes =
-        new ConcurrentHashMap<Point, List<RouteNote>>();
```

Comment:
This is fine because it is an example, but I don't think we want a lot more of PRs making removals swapping to the diamond when it doesn't matter. (You can quickly get into cases where it makes sense due to clarity or where it actually does matter on some versions of Java.)

Reply:
thanks, I have reverted it.